code/modules/hydroponics/spreading/spreading_response.dm

```diff
@@ -20,6 +20,8 @@
 /obj/effect/plant/Crossed(var/mob/living/victim)
```

Comment:
Mice and lizards get killed/deleted by vines with thorns when they cross them is the best reason I can come up with for why it was coded that way originally.

That whole bit probably could stand some clean-up, since it's kinda flakey.